docs/install/aws-install.md

```diff
@@ -53,30 +56,31 @@ After clicking the icon, the browser opens your AWS login page and directs you t
 6. **Additional EKS admin ARN (IAM user)** (o)&mdash;add an additional admin user to the instance. Users can be added after the stack has been created. For more information see [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
```

Comment:
1. **Additional EKS admin ARN (IAM user)** (o)&mdash;add an additional admin user to the instance. Users can be added after the stack has been created. For more information see [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).